models/movies/upcoming.js

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Everything here looks like duplicated code from `models/movies/search.js`. How can you remove this duplicated code?

Reply:
I should have created a class called 'getMovieCards' with these 2 methods.. and then create instances as i needed. This solution avoid this duplicated code.